### PR TITLE
Task3.1

### PR DESCRIPTION
**Pull request by uizyk**

### CODE REVIEW

**Comment by cherylli** [comment]
It looks unstyled to me, please check the latest figma file - the hifi-website, hifi-mobile for the latest design
image

File: src/components/RestaurantDetail/RestaurantDetail.jsx
Comment (uizyk) [inline src/components/RestaurantDetail/RestaurantDetail.jsx:1]: Refactored code to fit new component design provided by the team.

File: src/components/RestaurantDetail/RestaurantDetail.css
Comment (uizyk) [inline src/components/RestaurantDetail/RestaurantDetail.css:1]: Complete styling for desktop version, still need to add mediaqueries for mobile

**Review by cherylli — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**